packages/logging_cloudwatch/amplify_logging_cloudwatch/example/pubspec.yaml

```
@@ -0,0 +1,15 @@
+name: amplify_logging_cloudwatch_example
+description: An example app showcasing the amplify_logging_cloudwatch package.
+version: 1.0.0
+
```

I need to fix the template, but this should be here.

```suggestion
publish_to: none
```